pkg/services/publicdashboards/service/service_test.go

```diff
@@ -863,3 +867,212 @@ func TestDashboardEnabledChanged(t *testing.T) {
 		assert.True(t, publicDashboardIsEnabledChanged(&PublicDashboard{IsEnabled: false}, &PublicDashboard{IsEnabled: true}))
 	})
 }
+
+func TestPublicDashboardServiceImpl_ListPublicDashboards(t *testing.T) {
```

Comment:
I recently created a ticket for cleaning up service_test.go. Let's put this test in it's own file so it's easier to visualize and move about

Reply:
This func is testing method `ListPublicDashboards` from the service. Shouldn't belong to this file `service_test.go`?

Reply:
That's the problem EVERYTHING is in there. We need to start chopping this up.

Reply:
I will merge this PR and we can discuss this refactor on the issue you created.